scripts/bfasst/compare/waveform.py

```diff
@@ -74,41 +74,38 @@ def compare_netlists(self, design, runInterface):
             choice == 0
         ):  # Previous Status was unequivalent and User doesn't want to do any tests.
             return Status(CompareStatus.NOT_EQUIVALENT)
-        elif choice == 1:  # User wants to re-generate files.
+        if choice == 1:  # User wants to re-generate files.
             # Remove old testbenches and generate new ones
             for i in range(2):
                 if paths["tb"][i].exists():
                     paths["tb"][i].unlink()
             self.generate_files(multiple_files)
             if self.run_test():
                 return self.success_status
-            else:
-                return Status(CompareStatus.NOT_EQUIVALENT)
-        elif (
+            return Status(CompareStatus.NOT_EQUIVALENT)
+        if (
```

Comment:
Unnecessary parens

Reply:
Sorry, the if statement has unecessary parens.  Just so you're aware, the comments show up just below the relevant line, so I'm doing my best to put the comments right underneath the applicable section.